sdk/storage/inc/common/shared_key_policy.hpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "common/storage_credential.hpp"
+#include "http/policy.hpp"
+
+namespace Azure { namespace Storage {
+
+  class SharedKeyPolicy : public Core::Http::HttpPolicy {
+  public:
+    explicit SharedKeyPolicy(std::shared_ptr<SharedKeyCredential> credential)
+        : m_credential(std::move(credential))
+    {
+    }
+
+    ~SharedKeyPolicy() override {}
+
+    HttpPolicy* Clone() const override { return new SharedKeyPolicy(m_credential); }
```

Comment:
>HttpPolicy* Clone() const override { return new SharedKeyPolicy(m_credential); } [](start = 4, length = 80)

I think it would be best to add inline keyword for short implementation of member functions in header files.

Reply:
they are by default inline. So inline keyword is superfluous.